### ISSUE TITLE
Table3

### ISSUE THREAD

**Issue by ds-giao** [zhengzangw/DoPrompt#5]
Hello, your explanation in Table 3 states that the upper right corner is the distance between the domains, that is, the distance between the centroids of the domains. However, according to the data in the table, there is also a distance between the centroids of the same domain. For example, in the VIT-base table, the centroid distance between P and P is 23.5.

So I have a question: should the order of A P R C on the vertical axis be the same as the order of A C P R on the horizontal axis?

**Comment by zhengzangw** [comment]
Yes, you are correct. Seems a mistake. I think it should be ACPR.
